rust/src/transformations/dataframe/apply/mod.rs

```diff
@@ -14,6 +14,7 @@ use super::{DataFrame, DataFrameDomain};
 #[cfg(feature = "ffi")]
 mod ffi;
 
+#[deprecated(note = "Use Polars instead", since = "0.11.1")]
```

Comment:
Can these be changed to since 0.12? I think we shouldn't retrospectively deprecate.

Reply:
done! thanks.